app/mailers/inbox_mailer.rb

```diff
@@ -4,6 +4,6 @@ def admin_message(message, sender, receiver)
 		@message = message
 		@sender = sender
 		@receiver = receiver
-		mail to: 'dave.sloan@gmail.com', subject: 'Conversation between #{@sender.full_name} and from #{@receiver.full_name}'
+		mail to: 'dave.sloan@gmail.com', subject: 'BigTalker: A message has been sent between two users on BigTalker'
```

Comment:
Also @laurakathleen instead of put in the email 'dave.sloan@gmail.com', put ENV['BCC_MAIL']
So on development it will send to whatever email you set for BCC_EMAIL, like yours so you can check if you receive the email. On production, BCC_MAIL is already set to @slnwlf Dave email